internal/api/option/auth.go

```diff
@@ -272,10 +272,15 @@ func (a *authInterceptor) listServiceAccounts(
 			kubeclient.ServiceAccountsByGroupIndexField: group,
 		})
 	}
-	kargoNamespaces := make(map[string]struct{})
+	allowedNamespaces := make(map[string]struct{})
 	if a.cfg.KargoNamespace != "" {
 		// Kargo namespace is allowed by default
-		kargoNamespaces[a.cfg.KargoNamespace] = struct{}{}
+		allowedNamespaces[a.cfg.KargoNamespace] = struct{}{}
+	}
```

Comment:
We can get rid of this part now. Letting the admin who install specify allows them to name the kargo namespace as an allowed one if they want to, but we no longer have to hard code it.